batch/batch/cloud/azure/worker/worker_api.py

```diff
@@ -116,18 +116,16 @@ class AadAccessToken(LazyShortLivedToken):
     async def _fetch(self, session: httpx.ClientSession) -> Tuple[str, int]:
         # https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http
         params = {'api-version': '2018-02-01', 'resource': 'https://management.azure.com/'}
-        async with await request_retry_transient_errors(
-            session,
-            'GET',
+        resp_json = await retry_transient_errors(
+            session.get_return_json,
```

Comment:
I was also thinking about calling this `get_read_json`. Thoughts? I'm trying to convey a distinction between this method and the one which returns a body reading handle.